pkg/occlient/occlient.go

```diff
@@ -719,6 +738,19 @@ func (c *Client) CreateService(commonObjectMeta metav1.ObjectMeta, containerPort
 	return nil
 }
 
+// updateEnvVar updates the environmental variables to the container in the DC
+// dc is the deployment config to be updated
+// inputEnvVars is the array containing the corev1.EnvVar values
+func updateEnvVar(dc *appsv1.DeploymentConfig, inputEnvVars []corev1.EnvVar) error {
```

Comment:
I don't think we're testing this function?

Reply:
@cdrage Unit test coming up :D